docs/resources/restriction_policy.md

```diff
@@ -0,0 +1,56 @@
+---
+# generated by https://github.com/hashicorp/terraform-plugin-docs
+page_title: "datadog_restriction_policy Resource - terraform-provider-datadog"
+subcategory: ""
+description: |-
+  Provides a Datadog RestrictionPolicy resource. This can be used to create and manage Datadog restriction policies. See this page for more details https://docs.datadoghq.com/api/latest/restriction-policies/#supported-relations-for-resources.
+---
+
+# datadog_restriction_policy (Resource)
+
+Provides a Datadog RestrictionPolicy resource. This can be used to create and manage Datadog restriction policies. See this page for more details https://docs.datadoghq.com/api/latest/restriction-policies/#supported-relations-for-resources.
+
+## Example Usage
+
+```terraform
+# Create new restriction_policy resource
+
+
+resource "datadog_restriction_policy" "foo" {
+  bindings {
```

Comment:
@skarimo - same thing here, these are autogenerated right? also wondering why resource_id isn't getting added here after I made it required

Reply:
This is generated from the file: [examples/resources/datadog_restriction_policy/resource.tf](https://github.com/DataDog/terraform-provider-datadog/pull/1957/files/ccbe98b9b8919e4c94b5ebdb05931a17fd784d82#diff-76b94324cb4459196e393222837a2e0a992d09af0541b6ca21cb5c0cee57cabc)

So making changes to that and running `make docs` will reflect the changes here